orca-peering/orca-peering.gradle

```diff
@@ -18,6 +18,7 @@ apply from: "$rootDir/gradle/kotlin.gradle"
 apply from: "$rootDir/gradle/spock.gradle"
 
 dependencies {
+  api("com.netflix.spinnaker.keiko:keiko-core:$keikoVersion")
```

Comment:
keiko! What for?

Reply:
I needed the queue :( explanation forthcoming in our repo

Reply:
actually, removed...